index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>calculator</title>
+  </head>
+  <body>
+    <div>
+      <input type="text" id="screen" value="" currentOperation="" disabled />
```

Comment:
Why do we have a currentOperation attribute, it's not used anywhere

Reply:
Resolved